src/basic/rm-rf.h

```diff
@@ -32,10 +32,10 @@ int rm_rf_children(int fd, RemoveFlags flags, struct stat *root_dev);
 int rm_rf(const char *path, RemoveFlags flags);
 
 /* Useful for usage with _cleanup_(), destroys a directory and frees the pointer */
-static inline void rm_rf_and_free(char *p) {
+static inline void rm_rf_physical_and_free(char *p) {
         if (!p)
```

Comment:
Hm, I remove `tmpfs` *and* `physical`. So `rm_rf_physical_and_free` seems wrong. I'll rename `rm_rf_physical_and_free` to `rm_rf_and_free`